_rules/form-control-label-descriptive-cc0f0a.md

```diff
@@ -14,44 +14,59 @@ input_aspects:
   - DOM Tree
   - CSS Styling
 authors:
+  - Jean-Yves Moyen
+previous_authors:
```

Comment:
Did we decide to do this? Have we documented this?

Currently the site will not enumerate/display this.

Reply:
There has been an informal decision that previous authors should be credited when applicable, without a formal decision on how to implement it precisely.

I agree that this is currently not displayed.

I wanted to keep the information in an hopefully not to bad way until this is correctly tackled.

Issue for this is already opened: #799 (from the earlier version of this PR…) now moved to https://github.com/act-rules/act-rules-web/issues/35 in the website repo.